README.md

```diff
@@ -1,13 +1,17 @@
 # Community Extensions
-A collection of community extensions for the Phylum toolkit. These extensions
-expand Phylum's functionality even further than the [official extensions](https://github.com/phylum-dev/cli/tree/main/extensions).
-Some of these may be on their way to becoming official extensions. We hope you
-find them useful.
+
+A collection of community extensions for the Phylum toolkit.
+These extensions expand Phylum's functionality even further than the
+[official extensions](https://github.com/phylum-dev/cli/tree/main/extensions).
+Some of these may be on their way to becoming official extensions.
+We hope you find them useful.
```

Comment:
Is this just a rewrap? Seems a bit unnecessary, but if we're getting into it then...

1. This repo doesn't currently have an `.editorconfig` file. I'd suggest copying the file from CLI. That will at least settle the potential debate on line length.
2. If you're going to rewrap this paragraph, I'd recommend changing the link to a reference link (i.e., `[text][ref]`) so that the URL can be moved out of the paragraph

But I generally recommend avoiding rewraps to keep PRs small...

Reply:
It is just a rewrap. It will be reverted.